storage/redis/common/redis.go

```diff
@@ -0,0 +1,101 @@
+package common
```

Comment:
I think this code should just live in `github.com/chihaya/chihaya/storage/redis`. No other package is actually using this code.

Reply:
done